src/test/java/repository/UserRepositoryTest.java

```diff
@@ -66,8 +66,8 @@ void testFindAllUsers() throws SQLException {
 
         List<User> foundUsers = userRepository.findAll();
         assertThat(foundUsers)
-                .extracting(User::getId)
-                .containsExactly(1L, 2L);
+                .extracting(User::getId) //liste ID zwraca
```

Comment:
Shouldn't be here

Reply:
https://geek.justjoin.it/jak-dzialaja-map-filter-i-reduce-w-javascript 
extracting is like map (first paragraph)